apps/web/src/components/app/queue/QueueNowPlaying.tsx

```
@@ -10,7 +10,7 @@ export function QueueNowPlaying({ currentTrack }: QueueNowPlayingProps) {
   if (!currentTrack) return null;
 
   return (
-    <div className="space-y-3">
+    <div className="space-y-3" aria-label="Now Playing">
```

Biome (2.4.4)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)